packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -11,9 +11,20 @@ import {
 } from '@stencil/core';
 import classNames from 'classnames';
 import { SnackbarType } from '../types';
+import { hasSlotContent } from '../../util/component-utils';
 
 /**
  * Snackbars provide brief messages about app processes at the bottom of the screen. It functions as a wrapper around the material design's [Snackbar](https://github.com/material-components/material-components-web/tree/master/packages/mdc-snackbar) component
+ *
+ * #### Usage Notes:
+ *
+ * The `ino-snackbar` component can display an icon inside it, which can be customized as follows:
+ *
+ * - **Default Icon Behavior**: If no element is provided in the `icon-slot`, the component displays a default icon based on the `type` property. Each `type` corresponds to a specific `ino-icon`.
+ * - **Custom Icon (`<ino-icon>`)**: You can provide a custom `ino-icon` in the `icon-slot` to replace the default icon. Simply add `<ino-icon>` with your desired icon name to the `icon-slot`.
+ * - **Other Custom Elements (e.g., `<img>`)**: If you want to use a different element like an `<img>` as the icon, place it inside the `icon-slot`. This will override the default `ino-icon`. For example, `<img slot="icon-slot" src="your-icon-path" />`.
```

Comment:
Even though I like this explanation, I think it might be a bit too long just for the information that the slot can be used to add any additional element (icon, img, ...). For me personally, your explanation for the slot is enough.